lib/animas/animasSimulator.js

```diff
@@ -261,14 +265,23 @@ exports.make = function(config){
               }
             }
           }
+
+          // Filter out dates 2008 and earlier, as this is year 0 for Vibe.
+          // We are including 2008 as it's possible that the pump ran for a
```

Comment:
It says we are including 2008, but the code is `<= 2008` is dropped.  To me, I read this as 2008 is NOT included.  Or am I misreading the documentation.